Source/WebKit/Shared/API/glib/WebKitContextMenuPrivate.h

```diff
@@ -22,8 +22,23 @@
 #include "WebContextMenuItemGlib.h"
 #include "WebKitContextMenu.h"
 
+#if PLATFORM(GTK)
+#if USE(GTK4)
+#include <WebCore/GRefPtrGtk.h>
+#else
+#include <WebCore/GUniquePtrGtk.h>
+#endif
+#endif
```

Comment:
I'd prefer to write:

```suggestion
#if PLATFORM(GTK)
#include <WebCore/GRefPtrGtk.h>
#include <WebCore/GUniquePtrGtk.h>
#endif
```

When you are excessive with conditionalizing the includes, it becomes more likely that we'll accidentally break the build in the future.